pkg/systemd/services/service-details.jsx

```diff
@@ -572,12 +448,6 @@ export class ServiceDetails extends React.Component {
                                 <label className="control-label" htmlFor="path">{ _("Path") }</label>
                                 <span id="path">{this.props.unit.FragmentPath}</span>
                                 <hr />
-                                { this.props.originTemplate && this.props.isValid(this.props.originTemplate) &&
-                                    <>
-                                        <div />
-                                        <span>{_("Instance of template: ")}<a href={"#/" + this.props.originTemplate}>{this.props.originTemplate}</a></span>
```

Comment:
I'm a bit on the fence on this. Template instances are a thing, and it's important to show where they originate from. It should be clear from the name already though, and I suppose we don't have any particular information/status about the template itself (such as `systemctl cat`).

So after thinking aloud, I suppose I'm ok with this.

Reply:
yes, originally I kept this thing. Then I realized we show this only when the template is available on the system as link to that template (see the `this.props.isValid` part). That never succeeded once we stopped showing those templates so I just dropped it as a whole. We surely could just show that it was a template as a static information.